lib/consul-mesh-extension.ts

```diff
@@ -183,6 +202,9 @@ export class ECSConsulMeshExtension extends ServiceExtension {
         this.gossipEncryptKey = props.gossipEncryptKey;
         this.serviceDiscoveryName = props.serviceDiscoveryName;
         this.consulDatacenter = props.consulDatacenter || "dc1";
+        this.healthCheck = props.healthCheck;
+        this.consulChecks = props.consulChecks || undefined;
```

Comment:
I'm curious, can the user provide both `healthCheck` and `consulChecks` as well? If not, we should probably catch that!

Reply:
Actually they can provide both, just that consul has logic to accept only one at a time. if consul health checks are defined then they ignore ECS health checks. But that's a good question @upparekh . I will try to get an answer for when both the checks are defined what should happen?

Reply:
So as per discussion with HC we decided to throw an error when both(consul and ECS) the checks are mentioned.